.gitignore

```diff
@@ -5,4 +5,5 @@ client/build/
 common/build/
 ml-algorithms/build/
 plugin/build/
+plugin/src/test/resources/bwc/*
```

Comment:
Anywhere else using this path pattern? I don't see this in other places.

Reply:
This path is for storing the downloaded old and updated version of ml-commons plugins used in bwc test